assets/js/components/ReportError.js

```diff
@@ -67,7 +70,28 @@ export default function ReportError( { moduleSlug, error } ) {
 		purify.sanitize( message, { ALLOWED_TAGS: [] } )
 	);
 
-	return <CTA title={ title } description={ description } error />;
+	const retry = !! error?.selectorData?.storeName;
+
+	const handleRetry = useCallback( () => {
+		if ( retry ) {
+			const { selectorData } = error;
+			dispatch( selectorData.storeName ).invalidateResolution(
+				selectorData.name,
+				selectorData.args
+			);
+		}
```

Comment:
I don't think this check here is required, because this callback is only set when `retry` is truth-y. This can be removed, eg.

```suggestion
		const { selectorData } = error;
		
		dispatch( selectorData.storeName ).invalidateResolution(
			selectorData.name,
			selectorData.args
		);
```